lib/smart_answer_flows/am-i-getting-minimum-wage.rb

```diff
@@ -49,7 +51,7 @@ def define
         end
       end
 
-      use_shared_logic "minimum_wage"
+      append(Shared::MinimumWageFlow.build)
```

Comment:
Why have you used parenthesis in this method call? I'm interested because I think I would've naturally written `append Shared::MinimumWageFlow.build`.